src/index.ts

```diff
@@ -0,0 +1,10 @@
+export class Challenge {
+    buzz(x: number):Promise<string | number> {
+        return new Promise((resolve, reject) => {
+            if (x % 3 === 0 && x % 5 === 0) return resolve('fizzbuzz')
```

Comment:
That's ok but we prefer putting single line `if` blocks on a new line. Something like

```ts
if(condition)
    return true;
```